.werft/util/werft.ts

```diff
@@ -20,9 +20,11 @@ export class Werft {
     private tracer: Tracer;
     public rootSpan: Span;
     private sliceSpans: { [slice: string]: Span } = {}
-    public currentPhaseSpan: Span;
+    public currentPhase: string;
```

Comment:
The trace looks super weird at the moment, could it be related to this change?

![image](https://user-images.githubusercontent.com/24193764/161973426-98f0a814-ea36-4d5a-963f-7fcca5634db0.png)

Reply:
That is true but we decided to ignore that by just adding a warning.

Reply:
Could you also give me the links to the werft jobs?

Reply:
Sure, here is the latest one merged into main 🙂 
https://werft.gitpod-dev.com/job/gitpod-build-main.2819

Reply:
Just looked at this PR's latest (82 at the moment) and I don't see the same problem anymore 🤔 

https://werft.gitpod-dev.com/job/gitpod-build-wulfthimm-werft-ts-should-throw-8707.82/results

Reply:
What I basically did with the phase span was moving it from `this.currentPhaseSpan` to `this.phases[this.currentPhase].span`. This way it is not just ended deleted once the phase changes.

Reply:
I ran the job `with-vm=true` and `with-clean-slate-deployment=true` and I get problem with the missing span [[1](https://werft.gitpod-dev.com/job/gitpod-build-wulfthimm-werft-ts-should-throw-8707.84/results)].

Reply:
I just run a job from main, with the `with-vm` annotation as well.

You can see the problems with the logs (duplicated `observability` and `ensure VM readiness` slices), but the trace is completely fine.

Job: https://werft.gitpod-dev.com/job/gitpod-build-as-test-trace-from-main.1
Trace: https://ui.honeycomb.io/gitpod/datasets/werft/trace/o7JwMxxRLtP?span=18fec62887761c8d